### ISSUE TITLE
CoNLL 2012

### ISSUE THREAD

**Issue by HaniehP** [kermitt2/delft#29]
I have created the IOB2 format of CoNLL 2012 dataset based on the instruction you provided [here](https://github.com/kermitt2/delft/tree/master/delft/utilities).
However, I get the following error: 

TensorArray has size zero, but element shape [?,38] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.

which is due to having sentences of length one.
Any suggestions?

Thanks for your wonderful work!

**Comment by kermitt2** [comment]
Hello @HaniehP !
Thanks for the kind words.
I don't have this error with the assembled CoNLL 2012. Did you use Ontonotes version __5.0__?

Here are the stats for my assembled version:
```
Loading Ontonotes 5.0 CoNLL-2012 NER data...
82122 train sequences
	 nb. tokens 1633660
	 with nb. entities 239617
12678 validation sequences
	 nb. tokens 249271
	 with nb. entities 37534
8262 evaluation sequences
	 nb. tokens 152728
	 with nb. entities 20913
```

In principle the sentences of length one should not raise this kind of issues with the current version, it was fixed in #19.

**Comment by HaniehP** [comment]
Thanks for your reply. I exactly have the same stats for my assembled version.
**Fetch and Pull solved the issue.** Although, there are many empty lines in the generated files (eng.train, emg,test, eng.dev)
